Doc/using/configure.rst

```diff
@@ -146,6 +146,9 @@ recommended for best performance.
    Enable Profile Guided Optimization (PGO) using :envvar:`PROFILE_TASK`
    (disabled by default).
 
+   The C compiler ``clang`` requires ``llvm-profdata`` program for PGO. On
+   macOS, GCC also requires it: GCC is just an alis to clang on macOS.
```

Comment:
nit: alias
nit: Clang

Reply:
"Clang": oh no, I beg you pardon! It's now fixed :-D